### PR TITLE
Ruff q-z

### PR DESCRIPTION
**Pull request by bandophahita**

### CODE REVIEW

File: mypy.ini [hunk 1]
```diff
@@ -6,7 +6,7 @@ exclude = (?x)(
           | docs/
           )
 
-[mypy-screenpy.*]
+[mypy-screenpy_selenium.*]
```
Comment (perrygoy) [inline mypy.ini:9]: 🎉

File: screenpy_selenium/actions/clear.py [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
 from screenpy.pacing import beat
 from selenium.common.exceptions import WebDriverException
 
-from ..target import Target
+if TYPE_CHECKING:
+    from screenpy.actor import Actor
+
+    from ..target import Target
 
 SelfClear = TypeVar("SelfClear", bound="Clear")
```
Comment (perrygoy) [inline screenpy_selenium/actions/clear.py:16]: You can actually move these `TypeVar` declarations into the `TYPE_CHECKING` block, too, if you want. Then you can also import the `TypeVar` in there.

I don't have strong opinions on it, but i do like that all the type checking stuff is contained within that block that way.

**Review by perrygoy — APPROVED** [review]
Looks good to me! One minor comment that is not blocking. Thanks again!

File: screenpy_selenium/actions/clear.py (hunk 2 above)
Comment (bandophahita) [reply screenpy_selenium/actions/clear.py:16]: I'm about to yank all of those typevars out after the ruff changes.

**Pull request merged**